bundles/azure-service-broker-0.0.1/chart/azure-service-broker/charts/redis/README.md

```diff
@@ -0,0 +1,250 @@
+# Redis
+
+[Redis](http://redis.io/) is an advanced key-value cache and store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets, sorted sets, bitmaps and hyperloglogs.
+
+## TL;DR
```

Comment:
Please delete the whole content below. If you want to put emphasis on this content, we can add the following sentence instead:

`For more information on the Redis chart, see [this](https://github.com/helm/charts/tree/master/stable/redis) document.` 

In such case, do not apply the change of link in the previous comment.